### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by RAP2G**

### CODE REVIEW

File: frontend/src/components/Box.svelte [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
```
Comment (Albin-Rohde) [inline frontend/src/components/Box.svelte]: Is this used?

Comment (Albin-Rohde) [reply frontend/src/components/Box.svelte]: Nope not used, remove this file

File: frontend/src/api-client/user.ts [hunk 2]
```diff
@@ -121,6 +121,7 @@ export class User extends Client {
   }
 
   public async uploadAvatar(file: File): Promise<string> {
+    console.log(file);
```
Comment (axelburling) [inline frontend/src/api-client/user.ts]: remove this

File: frontend/src/components/Navbar.svelte [hunk 3]
```diff
@@ -1,76 +1,125 @@
...
+    Dropdown,
+    DropdownToggle,
+    DropdownMenu,
+    DropdownItem,
+  } from "sveltestrap";
+  import { User } from "../api-client";
+  import type { IUser } from "../api-client/types";
 
-import App from "../App.svelte";
+  const u = new User();
```
Comment (Albin-Rohde) [inline frontend/src/components/Navbar.svelte:13]: Why do we need a new blank user?

File: frontend/src/components/Feed.svelte [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  import type { IPost, IUser } from "../api-client";
+  import { Post } from "../api-client";
+  import type { PostSortString } from "../api-client/types";
+  export let feedType: PostSortString;
+  let post_description: string;
+  let post_title: string;
+  let posts: IPost[] = [];
+
+  (async () => {
+    console.log("hello");
```
Comment (axelburling) [inline frontend/src/components/Feed.svelte]: remove this

File: frontend/src/views/Dashboard/Dashboard.svelte [hunk 5]
```diff
@@ -1,19 +1,27 @@
 <script lang="ts">
-  import type { IPost, IUser } from "../../api-client";
+  import type { IUser } from "../../api-client";
```
Comment (Albin-Rohde) [inline frontend/src/views/Dashboard/Dashboard.svelte]: importing from same file multiple times

File: frontend/src/views/EditProfile/EditProfile.svelte [hunk 6]
```diff
@@ -0,0 +1,150 @@
+<script lang="ts">
+  import { User } from "../../api-client";
```
Comment (Albin-Rohde) [inline frontend/src/views/EditProfile/EditProfile.svelte:2]: multiple imports from same file

**Review by axelburling — CHANGES_REQUESTED** [review]

**Review by Albin-Rohde — CHANGES_REQUESTED** [review]

**Review by Albin-Rohde — APPROVED** [review]

**Review by axelburling — APPROVED** [review]
LGTM!

**Pull request merged**